wazo_calld/plugins/applications/services.py

```diff
@@ -417,7 +417,7 @@ def get_value():
                 return
 
             logger.debug('waiting for a setvar to complete')
-            time.sleep(0.001)
+            time.sleep(0.01)
```

Comment:
Do you want to edit commit message?

Reply:
what do you mean?

Reply:
the commit message for this change is:

```
    set channel var sync: wait up to 0.2 seconds
    
    the old value waited a maximum of 0.02 seconds which was not long enough in
    some cases
```

Reply:
Oh nvm, I've just don't have context lines to see the loop :sweat_smile: